scene/3d/baked_lightmap.cpp

```diff
@@ -1238,6 +1272,15 @@ BakedLightmap::BakeQuality BakedLightmap::get_capture_quality() const {
 	return capture_quality;
 }
 
+void BakedLightmap::_set_preview_bake(bool p_preview_bake) {
+	preview_bake = p_preview_bake;
+	update_configuration_warning();
+}
+
+bool BakedLightmap::_is_preview_bake() const {
+	return preview_bake;
+}
+
```

Comment:
I would move the "preview_bake" flag to `BakedLightmapData`. I think it makes more sense to mark the resource as "baked in preview mode", rather than setting the flag in the node itself.  Since lightmap data is saved in files by default, a user could load a different `BakedLightmapData` resource and the node would still be flagged as using a preview bake.

Maybe we could even add a `.preview.` infix to the saved file name, so baking in preview mode doesn't overwrite a "final" bake you may already have.